### PR TITLE
GitLab Pipelines Edit

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Add a docs link to the Logs Processors doc, in-app links to the Pipeline Setup page.

Chat with Kassen on #support-ci-visibility in Slack.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 1]
```diff
@@ -238,19 +238,20 @@ See the table below for the message and domain correlated with each error type.
...
+
 * GitLab.com (SaaS)
-* GitLab >= 14.8 (self-hosted) only if using [object storage to store job logs][7]
+* GitLab >= 14.8 (self-hosted) only if you are using [object storage to store job logs][7]
 
 To enable collection of job logs:
 
-1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This reveals the `Enable logs collection` option in the Datadog integration.
-2. Enable the `Enable logs collection` option and save the changes.
+1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This allows you to see the **Enable job logs collection** button on the [Pipeline Setup page][17].
```
Comment (AdrianLC) [inline content/en/continuous_integration/pipelines/gitlab.md:247]:
```suggestion
1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This allows you to see the **Enable job logs collection** check box on the [Pipeline Setup page][17].
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 2]
```diff
@@ -272,3 +273,6 @@ Log files larger than 1GiB are truncated.
 [14]: /continuous_integration/pipelines/gitlab/?tab=gitlabcom#correlate-infrastructure-metrics-to-jobs
 [15]: /continuous_integration/pipelines/gitlab/?tab=gitlabcom#view-error-messages-for-pipeline-failures
 [16]: /account_management/teams/
+[17]: https://app.datadoghq.com/ci/setup/pipeline?provider=gitlab
+[18]: https://app.datadoghq.com/ci/pipeline-executions?query=ci_level%3Apipeline%20%40ci.provider.name%3Agitlab&index=cipipeline
```
Comment (AdrianLC) [inline content/en/continuous_integration/pipelines/gitlab.md:277]: This won't get the user to anything where they can see logs or pipelines that have logs exclusively. I don't think we have a deep link for this

Comment (alai97) [reply content/en/continuous_integration/pipelines/gitlab.md:277]:
```suggestion
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 3]
```diff
@@ -238,19 +238,20 @@ See the table below for the message and domain correlated with each error type.
...
 
 To enable collection of job logs:
 
-1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This reveals the `Enable logs collection` option in the Datadog integration.
-2. Enable the `Enable logs collection` option and save the changes.
+1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This allows you to see the **Enable job logs collection** button on the [Pipeline Setup page][17].
+2. Click **Enable job logs collection** and click **Save changes**.
 
-Job logs are collected in the [Logs][9] product and automatically correlated with the GitLab pipeline within CI Visibility.
+Job logs are collected in [Log Management][9] and are automatically correlated with the [GitLab pipeline][18] in CI Visibility. Log files larger than 1GiB are truncated.
```
Comment (alai97) [inline content/en/continuous_integration/pipelines/gitlab.md:250]:
```suggestion
Job logs are collected in [Log Management][9] and are automatically correlated with the GitLab pipeline in CI Visibility. Log files larger than 1GiB are truncated.
```

**Review by AdrianLC — APPROVED** [review]

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 4]
```diff
@@ -238,19 +238,20 @@ See the table below for the message and domain correlated with each error type.
...
 
 To enable collection of job logs:
 
-1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This reveals the `Enable logs collection` option in the Datadog integration.
-2. Enable the `Enable logs collection` option and save the changes.
+1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This allows you to see the **Enable job logs collection** check box on the [Pipeline Setup page][17].
+2. Click **Enable job logs collection** and click **Save changes**.
 
-Job logs are collected in the [Logs][9] product and automatically correlated with the GitLab pipeline within CI Visibility.
+Job logs are collected in [Log Management][9] and are automatically correlated with the GitLab pipeline in CI Visibility. Log files larger than 1GiB are truncated.
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/gitlab.md:250]:
```suggestion
Job logs are collected in [Log Management][9] and are automatically correlated with the GitLab pipeline in CI Visibility. Log files larger than one GiB are truncated.
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 5]
```diff
@@ -272,3 +273,5 @@ Log files larger than 1GiB are truncated.
 [14]: /continuous_integration/pipelines/gitlab/?tab=gitlabcom#correlate-infrastructure-metrics-to-jobs
 [15]: /continuous_integration/pipelines/gitlab/?tab=gitlabcom#view-error-messages-for-pipeline-failures
 [16]: /account_management/teams/
+[17]: https://app.datadoghq.com/ci/setup/pipeline?provider=gitlab
+[19]: /logs/log_configuration/processors/
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/gitlab.md:277]:
```suggestion
[18]: /logs/log_configuration/processors/
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 6]
```diff
@@ -238,19 +238,20 @@ See the table below for the message and domain correlated with each error type.
...
+2. Click **Enable job logs collection** and click **Save changes**.
 
-Job logs are collected in the [Logs][9] product and automatically correlated with the GitLab pipeline within CI Visibility.
+Job logs are collected in [Log Management][9] and are automatically correlated with the GitLab pipeline in CI Visibility. Log files larger than 1GiB are truncated.
 
-<div class="alert alert-info"><strong>Note</strong>: Logs are billed separately from CI Visibility</div>
+<div class="alert alert-info"><strong>Note</strong>: Logs are billed separately from CI Visibility.</div>
 
-Log files larger than 1GiB are truncated.
+For more information about processing job logs collected from the GitLab integration, see the [Processors documentation][19].
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/gitlab.md:254]:
```suggestion
For more information about processing job logs collected from the GitLab integration, see the [Processors documentation][18].
```

File: content/en/continuous_integration/pipelines/gitlab.md [hunk 7]
```diff
@@ -238,19 +238,20 @@ See the table below for the message and domain correlated with each error type.
...
+
 * GitLab.com (SaaS)
-* GitLab >= 14.8 (self-hosted) only if using [object storage to store job logs][7]
+* GitLab >= 14.8 (self-hosted) only if you are using [object storage to store job logs][7]
 
 To enable collection of job logs:
 
-1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This reveals the `Enable logs collection` option in the Datadog integration.
-2. Enable the `Enable logs collection` option and save the changes.
+1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This allows you to see the **Enable job logs collection** check box on the [Pipeline Setup page][17].
```
Comment (buraizu) [inline content/en/continuous_integration/pipelines/gitlab.md:247]:
```suggestion
1. Enable the `datadog_integration_logs_collection` [feature flag][8] in your GitLab self-hosted or GitLab.com account. This allows you to see the **Enable job logs collection** checkbox on the [Pipeline Setup page][17].
```

Comment (buraizu) [reply content/en/continuous_integration/pipelines/gitlab.md:247]: `checkbox` looks to be much more frequently used in the documentation site. This is also preferred in some style guides:
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/c/check-checkbox-check-mark

**Review by buraizu — COMMENTED** [review]
Thanks, left some feedback!

**Review by buraizu — APPROVED** [review]
Thanks for making those updates, LGTM!

**Pull request merged**